plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPath.java

```diff
@@ -73,12 +74,13 @@ static void addLibraries(IProject project, List<Library> libraries, IProgressMon
     }
     javaProject.setRawClasspath(newRawClasspath, monitor);
   
-    runContainerResolverJob(javaProject);
+    IEclipseContext context = EclipseContextFactory.getServiceContext(
+        FrameworkUtil.getBundle(classFromBundle).getBundleContext());
+    
+    runContainerResolverJob(javaProject, context);
   }
   
-  private static void runContainerResolverJob(IJavaProject javaProject) {
-    IEclipseContext context = EclipseContextFactory.getServiceContext(
-        FrameworkUtil.getBundle(CreateAppEngineStandardWtpProject.class).getBundleContext());
+  private static void runContainerResolverJob(IJavaProject javaProject, IEclipseContext context) {
```

Comment:
I can certainly do that. As currently written, this move is a fairly safe refactoring. However if I don't pass the class, then we're changing (I think) which bundle is used which does not seem to be a no-op. As long as that's OK, I can do it.